README.rst

```diff
@@ -87,14 +87,12 @@ Check out the `Getting Started`_ guide for more information.
 .. _Getting Started: https://beets.readthedocs.org/page/guides/main.html
 .. _software repositories: https://repology.org/project/beets/versions
 
-Contribute
+Contributing
```

Comment:
I know this is a little silly, but the rest of the headings in the README are all imperative-mood verbs, so I kind of liked the aesthetic of letting this one stay the same way too. :smiley: